servers/serverserial.js

```diff
@@ -253,7 +253,7 @@ class ServerSerial extends EventEmitter {
         // remember open sockets
         modbus.socks = new Map();
 
-        modbus._server.on("open", function() {
```

Comment:
Hi, can you give some description what this do, and why this change is needed ?

Reply:
The `open` event never gets emitted.
I think it may be because of how `ServerSerialPipeHandler` works? 
But the `open` event from `SerialPort`  isn't piped and so you never receive an `initialized` event either.